app/graphql/types/host.rb

```diff
@@ -0,0 +1,12 @@
+module Types
+  class Host < GraphQL::Schema::Object
```

Comment:
Should be named `HostType` to keep consistency with already created `ModelType`

Reply:
This will be consistent after a rebase. I have decided to drop the `Type` suffix everywhere.